HW/aboutme.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Me</title>
+</head>
+
+<body>
+    <link rel="stylesheet" href="styleaboutme.css">
+    <h2>
+        <a href="Aboutme.html">About Me</a> | <a href="Portfolio.html">Portfollo</a>
+    </h2>
+
+    <h1>Joe Johnson </h1>
+
+    <div>
+
+        <h3>I ENJOY LIFE AS A DEVELOPER</h3>
+
+        <p>
+            I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in<br>
+            the graphic design world, specializing in the creation of responsive websites.
+        </p>
+
+        <img src="profile pic.png" alt=" profile pic" />
```

Comment:
try to not use spaces when naming documents or pictures. Instead, rename it "profile_pic.png"